pom.xml

```diff
@@ -12,15 +12,58 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.release>17</maven.compiler.release>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>33.3.1-jre</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.11</version>
+      <groupId>org.project-kessel</groupId>
+      <artifactId>relations-client-java</artifactId>
+      <version>0.8</version>
+    </dependency>
+    <dependency>
+      <groupId>com.nimbusds</groupId>
+      <artifactId>oauth2-oidc-sdk</artifactId>
+      <version>11.13</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
+      <version>4.28.2</version>
```

Comment:
Noticed you are using protobuf v4.28.2. I'm wondering if this may cause problems like we saw with protovalidate-java moving to v4 while our generation is on v3?

Reply:
Good call! In this case we're just using the utils for `Jsonformat` to convert json to proto objects, but it still makes sense to match all these library versions to the latest protoc version that supports java code generation.